Tasks/AzureResourceGroupDeploymentV2/operations/ResourceGroup.ts

```diff
@@ -515,7 +516,7 @@ export class ResourceGroup {
         });
     }
 
-    private async performAzureDeployment(armClient: armResource.ResourceManagementClient, deployment: Deployment): Promise<void> {
+    private async performAzureDeployment(armClient: armResource.ResourceManagementClient, deployment: Deployment, retryCount = 0): Promise<void> {
         if (deployment.properties["mode"] === "Validation") {
```

Comment:
Also, if you were able to repro the issue, then were you able to verify that the retry logic is required at the template deployment, and not the resource group creation?

Reply:
Yes retry logic make sense at template deployment only not during the resource group creation time because there is provision for Strongly Consistent Write operation